### DISCUSSION TITLE
如何修改自带的动画速度？

### DISCUSSION THREAD

**Discussion by zms9110750** [BlazorComponent/MASA.Blazor#1228] [Q&A] [answered]
我把进度-圆环-动画的代码抄下来，改成了每秒执行10次，每次将数值+1。他的动画虽然流畅，但延迟太高。如果我用它做一个放置类游戏，那他的进度应该时刻保持和数值匹配。

**Comment by doddgu** [comment]
@capdiem PTAL

**Comment by capdiem** [comment, marked as answer]
@zms9110750 进度-圆环-动画 在哪？

**Reply by zms9110750** [reply]
这个示例：https://github.com/masastack/MASA.Blazor/blob/main/docs/Masa.Blazor.Docs/Examples/components/progress-circular/Rotate.razor
我把执行间隔改成10毫秒他就跟不上了。但是等比放大间隔和变化值，他却能跟上。说明有个开始停顿或结束停顿。

**Reply by capdiem** [reply]
![image](https://user-images.githubusercontent.com/16507773/232050575-657384b2-7549-4964-932e-df14509ed259.png)

这里改成 0.06s，默认是 0.6s。

**Reply by capdiem** [reply]
如果你的间隔是0.1s执行一次，那css的过渡动画也需要小于0.1s。

**Comment by doddgu** [comment]
这个QA先关闭了，如果仍然有问题可以reopen
